var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -107,6 +109,12 @@ class Gromacs(CMakePackage, CudaPackage):
     )
     variant("opencl", default=False, description="Enable OpenCL support")
     variant("sycl", default=False, when="@2021:", description="Enable SYCL support")
+    variant(
+        "sycl",
+        default=True,
+        when="@2021: +rocm",
+        description="Enable SYCL support when using ROCm",
```

Comment:
I don't think this description is correct.

1. We don't "Enable SYCL support when using ROCm". We use SYCL, which, in turn, uses ROCm.
2. ACpp/ROCm is only supported since `@2022:`.

Reply:
Fixed it, thanks.